### DISCUSSION TITLE
how can scrap all sitemap location at a time?

### DISCUSSION THREAD

**Discussion by era-satish** [eliasdabbas/advertools#229] [Q&A] [answered]
how can scrap all sitemap location a time?

**Comment by eliasdabbas** [comment, marked as answer]
What exactly do you mean? 
What is the input data that you have, and what kind of output do you want?

**Reply by era-satish** [reply]
![Capture](https://user-images.githubusercontent.com/111265451/184612070-a4d6a049-3bae-4f2f-9bbf-41a63c05b320.PNG)
I'm trying to view 10000 loc but it is showing upto 1000 at a time

**Reply by eliasdabbas** [reply]
That's a limitation that can be changed in the notebook settings. The full list is still available, and you can loop through them, or select any URL at any index

**Reply by era-satish** [reply]
where is notebook setting

**Reply by era-satish** [reply]
can you change this setting on your demo site

**Reply by eliasdabbas** [reply]
This notebook is for demonstration purposes only. 
In any case why would you need to see 1000 rows at a time? You will have to scroll anyways, so you can extract the subsets you want.
